test/tools/ossfuzz/abiV2ProtoFuzzer.cpp

```diff
@@ -50,11 +74,37 @@ DEFINE_PROTO_FUZZER(Contract const& _input)
 		// We always call the function test() that is defined in proto converter template
 		hexEncodedInput = methodIdentifiers["test()"].asString();
 	}
-	catch (...)
+	catch (Exception const&)
 	{
-		cout << contract_source << endl;
-		throw;
+		return;
 	}
-	// TODO: Call evmone wrapper here
-	return;
+
+	// We target the default EVM which is the latest
+	langutil::EVMVersion version = {};
+	auto hostContext = EVMHost(version, evmone);
+	dev::bytes input = fromHex(hexEncodedInput);
+	evmc_message message;
+	message.kind = EVMC_CALL;
+	// FIXME: We call a function in one contract that calls a function in another.
+	// So, call depth = 2?
+	message.depth = 2;
+	// Call value is zero
+	message.value = {0};
+	message.gas = std::numeric_limits<int64_t>::max();
+	message.input_data = input.data();
+	message.input_size = input.size();
+
+	// Execute byte code on evm one
+	auto result = evmone.execute(
```

Comment:
Better use `hostContext.call(message)`

Reply:
tbh I don't understand what `hostContext.call(message)` does. What should the `message.input_data` field be to call a contract function?